Assets/HoloToolkit/Input/Scripts/InputSources/EditorHandsInput.cs

```diff
@@ -326,7 +387,7 @@ private void SendHandVisibilityEvents()
             // Send event for new hands that were added
             foreach (uint newHand in newHands)
             {
-                inputManager.RaiseSourceDetected(this, newHand);
+                InputManager.Instance.RaiseSourceDetected(this, newHand);
```

Comment:
I actually have a suggestion about making all the `InputManager` members static to remove the `.Instance` verbosity (https://github.com/Microsoft/HoloToolkit-Unity/issues/534)